SimonOkello/Questioner
# [ft #163080788] Modify README.md file

**What does this PR do?**
Modifies `README.md`

**Description of Task to be completed?**
Have the following achieved:
The `README.md` file should have TravisCI and Coveralls badge
The `README.md` file should have API description

**How should this be manually tested?**
Clone this repo:
`https://github.com/SimonOkello/Questioner.git`
`cd Questioner`
Scroll down to see `README.md`
![screenshot from 2019-01-11 13-13-42](https://user-images.githubusercontent.com/28682748/51027766-c1186f00-15a2-11e9-8c3b-47b49eb85094.png)

**Screenshot**
